languages/scala/exercises/concept/basics/LasagnaTest.scala

```diff
@@ -0,0 +1,27 @@
+/** @version 1.3.0 */
+class LasagnaTest extends FunSuite with Matchers {
+
+  test("expected minutes in oven") {
+    new Lasagna().expectedMinutesInOven() should be(40)
+  }
+
+  test("remaining minutes in oven after 25 minutes") {
+    new Lasagna().remainingMinutesInOven(25) should be(15)
```

Comment:
To encourage students to use TDD when solving an exercise, we skip all but the first test by default (see [the test suite of a v2 Scala exercise](https://github.com/exercism/scala/blob/master/exercises/acronym/src/test/scala/AcronymTest.scala#L11)). This would look like this:

```suggestion
    pending
    new Lasagna().remainingMinutesInOven(25) should be(15)
```

This comment also applies to the subsequent tests in this file.

Reply:
Thanks! I wanted to add those after finalizing the tests and forgot it again :)